### ISSUE TITLE
[COGNITION, SCREENREADER, KEYBOARD] Hiding 112 characters in a 40 character show/hide introduces screenreader and keyboard extra effort

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#5861]
## Description

Content hidden in the "Learn more about military base addresses" additional information component is 112 characters, whereas the link itself is 40 characters. To reveal the content invariably takes some motor and cognitive effort on the part of our user. 

For use cases with keyboard, screen reader, mobility, and/or a range of cognitive conditions, a person must trigger the show/hide, hear text repeated, close it, and then hear it repeated again. The content is also hidden from the accessibility tree until triggered, requiring an assistive tech notice, triggering the repeated content. Since this hidden content is relatively short, best practice recommends not hiding it. 

Additionally, the connection between the content and the checkbox is not clear for non-sighted users who cannot see the visual affordances.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

- As a screen reader user, I want to minimize the interruptions and repetition of text so I may digest the information.
- As a person with limited hand strength or mobility or cognitive considerations, I want to minimize the clicks necessary to digest information.

## Steps to Recreate

1. Enter `https://staging.va.gov/profile/` in browser
2. Start screenreading device of choice (JAWS, NVDA, or VoiceOver)
3. Navigate to the show/hide link for "Learn more about military base addresses"
4. Toggle to reveal, toggle to hide
5. Verify the label is repeated
6. Verify it is a small amount of text, not much more than the amount of space as the show/hide trigger alone.

## Possible Fixes (optional)

1. **Consider** putting the content currently hidden in the additional info component on the screen without the need to show/hide.

2. Add to the input type=”checkbox, aria-describedby=”id for the sentence 'The United States is auto…'" so that screen readers create the contextual connection.
```html
<div class="form-checkbox schemaform-first-field">
  <input type="checkbox" id="root_view:livesOnMilitaryBase" name="root_view:livesOnMilitaryBase" aria-describedby=”unique-id">
  <label class="schemaform-label" for="root_view:livesOnMilitaryBase">
    I live on a United States military base outside of the United States
  </label>
  <p id=”unique-id">
    The United States is automatically chosen as your country if you live on a military base outside of the country.
  </p>
</div>
```

## WCAG or Vendor Guidance (optional)

* [If something is relevant to the page, then it's probably relevant to the user too, so I'd recommend showing it to the user.](https://www.seroundtable.com/google-content-hiding-19495.html)

* [Article on show/hide content ux](https://www.learningtoo.eu/articles/show-and-hide-content.htm#swot)

## Screenshots or Trace Logs

Content is not available to the dom or accessibility tree, until it is triggered
![authdex-addressvalidation-hiddenfroma11ytree](https://user-images.githubusercontent.com/57469/74472479-ff502400-4e6f-11ea-97eb-aeb841e0de94.png)

**Comment by Samara-Strauss** [comment]
This needs some design consideration and does not immediately affect the address validation launch, so I'm going to put this in our backlog.

**Comment by mshea0606** [comment]
@tressaellen link to new approach

**Comment by Samara-Strauss** [comment]
[This is the updated design](https://github.com/department-of-veterans-affairs/va.gov-team/issues/11043#issuecomment-666655878). I think we might just be waiting for sign off from Peggy.
